James branch

Mrwhitepantz opened the pull request:
Added room generation part of maze generation, need to add corridors still

Mrwhitepantz committed:
Rearranged some files

also made mazegenerator files

Mrwhitepantz committed:
Update ForestMaze.cs

Mrwhitepantz committed:
merge main

Mrwhitepantz committed:
ForestMaze rooms clearings working

the forest maze generator successfully creates clearings

Mrwhitepantz committed:
Forest maze gen debugging

debugged issues with room placement/generation inside forest mazes. commented mazegeneration out until can add connection between rooms

AustinSickles reviewed (changes requested):
Can you download the main brance again and resubmit this, I did not know this pull was up and now there are other changes made after this.

Mrwhitepantz committed:
Forest Mazes Working

Forest mazes connect clearings and exits to create fully reachable cells that all connect with adjacent cells

Mrwhitepantz committed:
Merge Main

main merge
